src/components/Game/Wordsy/Utils/Modals/EndGameModal.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
```

Comment:
Let's find some time to have a chat about directory structure and some strategies for games directory structure in particular. 
For now if you could move this file into the Wordsy root directory to avoid the extra nesting that would be great. Thanks.

Reply:
Sounds good. I'll update the structure and plan a discussion around this.
Thanks.